### ISSUE TITLE
请问这个控制面板能实现多个站点同步吗？

### ISSUE THREAD

**Issue by shun556618** [webmin/webmin#1920]
例如：在A服务器中发布内容后，B服务器同步A发布中内容。主要是想实现站点之间的内容同步。

**Comment by chris001** [comment]
> Can this control panel realize multiple site synchronization?
> For example: after the content is published on server A, server B synchronizes the content published by A. Mainly to achieve content synchronization between sites

**Comment by jcameron** [comment]
Yes, this is possible using Cloudmin, which can setup syncing between one master Virtualmin system and multiple replicas.

**Comment by Galdlsys** [comment]
> Yes, this is possible using Cloudmin, which can setup syncing between one master Virtualmin system and multiple replicas.

Excuse me, is this data synchronization A to B or B to A? In addition, can we achieve database synchronization?

**Comment by jcameron** [comment]
Can you tell us more about what kind of setup you're looking to achieve exactly?

**Comment by Galdlsys** [comment]
Suppose there are two servers, A and B. A is the primary server and B is the secondary server. webmin is installed on both servers. At the same time, the same website content is created in these panels with different domain names.
Suppose that new content is published on the website of server A, and the website of server B needs to synchronize the content published by server A, including files, databases, etc., and server B needs to conduct detection synchronization on a daily or hourly basis or in real time.
In addition, if it can be realized, I want to know whether A actively pushes the content to B or B pulls A's data.

**Comment by Galdlsys** [comment]
> Can you tell us more about what kind of setup you're looking to achieve exactly?

Suppose there are two servers, A and B. A is the primary server and B is the secondary server. webmin is installed on both servers. At the same time, the same website content is created in these panels with different domain names.
Suppose that new content is published on the website of server A, and the website of server B needs to synchronize the content published by server A, including files, databases, etc., and server B needs to conduct detection synchronization on a daily or hourly basis or in real time.
In addition, if it can be realized, I want to know whether A actively pushes the content to B or B pulls A's data.

**Comment by jcameron** [comment]
AH ... so we do support synchronization, but only for domains with the same name. The idea is to allow users to have multiple machines hosting replicas of their site, to spread the load and add redundancy. 

What is the use case for doing this with different domain names?
